portfolio/src/main/webapp/index.html

```diff
@@ -221,10 +221,9 @@ <h3>arrington's friends comments</h3>
                 <button name="up" id="up" type="submit" onclick="changeButtonValDown()">&laquo;</button>
```

Comment:
Having these buttons submit the form AND have an onclick callback is a bit confusing. I can't tell which one happens first. Can you remove these from the form if it isn't needed.

Reply:
ok, i put the submit in the javascript function